src/functions/pollUsers/domain/examiner-cache-reconciler.ts

```diff
@@ -1,9 +1,23 @@
-import { cacheStaffNumbers, uncacheStaffNumbers } from '../framework/repo/dynamodb/cached-examiner-repository';
+import {
+  uncacheStaffNumbers, cacheStaffDetails,
+} from '../framework/repo/dynamodb/cached-examiner-repository';
+import { StaffDetail } from '../../../common/application/models/staff-details';
 
-export const reconcileActiveAndCachedExaminers = async (activeStaffNumbers: string[], cachedStaffNumbers: string[]) => {
-  const staffNumbersToCache = activeStaffNumbers.filter(staffNumber => !cachedStaffNumbers.includes(staffNumber));
-  await cacheStaffNumbers(staffNumbersToCache);
+export const reconcileActiveAndCachedExaminers
+= async (activeStaffDetails: StaffDetail[], cachedStaffNumbers: string[]) => {
```

Comment:
I think elsewhere in the DES codebase, we tend to use this style when breaking a function declaration over multiple lines:
```
export const reconcileActiveAndCachedExaminers = async (
  activeStaffDetails: StaffDetail[],
  cachedStaffNumbers: string[],
) => {
  ...
}
```